test-app/__tests__/webpack.config.babel.js

```diff
@@ -0,0 +1,11 @@
+import test from 'ava'
+import config from '../webpack.config.babel'
+
+test('it creates the config file', t => {
+  const plugins = config.plugins
+  delete config.plugins // because it has absolute paths and bring a lot of bloat
```

Comment:
Is this OK?
Or do you know solution for this, if not I'll take a deeper look.

Reply:
Probably we can do another check for plugins, at least that all plugins are present.

Reply:
I think I was a bit too eager to send a PR yesterday.
After giving this a second look I believe that a snapshot test might not be needed because: 
- each block should have its own separate test
- the `createConfig` has its own test

So maybe a sanity check to check it isn't throwing any errors should be enough?